rca/utils/models.py

```diff
@@ -429,3 +430,29 @@ class Meta:
             [FieldPanel("disable_apply_tab")], "Global settings for programme pages"
         )
     ]
+
+
+@register_snippet
+class ShortCourseDetailSnippet(models.Model):
+    SHORT_COURSE_DETAIL_TYPES = [(1, "FAQs"), (2, "T&Cs")]
```

Comment:
Separating out the choices values into separate constants means you don't have to repeat the value below:

```
FAQ = 1
T&C = 2
SHORT_COURSE_DETAIL_TYPES = [(FAQ, "FAQs"), (T&C, "T&Cs")]

def body(self):
        if self.snippet_type == FAQ:
            ...
```

More info https://www.b-list.org/weblog/2007/nov/02/handle-choices-right-way/

Reply:
:+1: